libraries/WiFiS3/src/WiFi.h

```diff
@@ -19,24 +19,43 @@
 
 #define WIFI_FIRMWARE_LATEST_VERSION "0.4.1"
 
+#define WL_MAX_AP_LIST 10
```

Comment:
I would move `WL_MAX_AP_LIST` to WiFi.cpp file and rename it to something like `WIFI_MAX_SSID_COUNT` adding the possibility to override it

```
#ifndef WIFI_MAX_SSID_COUNT
  #define WIFI_MAX_SSID_COUNT 10
#endif
```